docs/pages/documentation/3-querying-with-graql/one-big-file.md

```diff
@@ -0,0 +1,100 @@
+---
+title: One Big File (Obviously change this)
+keywords: graql, query
+last_updated: April 2017
+tags: [graql]
+summary: "TODO"
+sidebar: documentation_sidebar
+permalink: /documentation/graql/one-big-file.html
+folder: documentation
+---
+
+## Query
+
+A [query](#query) is an action, in some cases preceded by a [match](#match). Examples of actions are
+[define](#define-query), [undefine](#undefine-query), [get](#get-query), [insert](#insert-query) and
+[delete](#delete-query).
+
+## Pattern
+
+A [pattern](#pattern) is either a [variable pattern](#variable-pattern), a [conjunction](#conjunction) or a
+[disjunction](#disjunction).
+
+### Variable pattern
+
+A [variable pattern](#variable-pattern) is a [variable](#variable) followed by zero or more [properties](#property)
+(optionally separated by commas `,`) and ending in a semicolon `;`.
+
+### Conjunction
+
+A [conjunction](#conjunction) is one or more [patterns](#pattern) surrounded by curly braces `{ }` and ending in a
```

Comment:
That does not really explain what a conjunction is, just how you write it. Probably both are needed

Reply:
Of course it applies to the other things, as well

Reply:
Yeah, definitely! I wanted to get the structure of the things right first.